combat-trainer.lic

```diff
@@ -3670,6 +3670,7 @@ end
 before_dying do
   DRCA.release_cyclics
   DRCA.shatter_regalia?
+  fput('close fissure') if DRStats.warrior_mage? && DRRoom.room_objs.any? { |x| /fissure/ =~x }
```

Comment:
❓ Why is this change necessary?

https://github.com/rpherbig/dr-scripts/blob/bb98adbace693692e4a9e386b20dff1e19c75774/data/base-spells.yaml#L915 should close any open fissure before casting a new one, right?